helm-frontend/src/types/global.d.ts

```diff
@@ -2,5 +2,5 @@ interface Window {
   RELEASE: string;
   SUITE: string;
   BENCHMARK_OUTPUT_BASE_URL: string;
-  HELM_TYPE: string;
+  RELEASE_INDEX_ID: string;
```

Comment:
for naming consistency (previous name is reasonable, and we can rename in the other direction if desired)